examples/ensure-name-substring/imperative/README.md

```diff
@@ -0,0 +1,25 @@
+# ensure-name-substring: Imperative Example
+
+### Overview
+
+This example demonstrates how to imperatively invoke the [`ensure-name-substring`]
+function to prepend prefix in the resource names.
+
+### Function invocation
+
+Get the config example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/ensure-name-substring/imperative
+kpt fn eval imperative --image=gcr.io/kpt-fn/ensure-name-substring:unstable -- prepend=prod
```

Comment:
I had a comment in one of the previous fishfoods about the example package naming. You actually want to quality the package name with the function:

`ensure-name-substring-imperative` or something like that. Otherwise, when you fetch a package, there's name collision on the local filesystem (Everything is called `simple` and you have to delete otherse, etc.)

Reply:
Yes, that's my plan. I will change examples directory names and change the workflow to fetch package first.
I will do it in a separate PR since they are mostly mechanical changes.

Reply:
Documentation in kpt repo needs to be updated as well but I guess since everything should be versioned, it doesn't have to be in lock step. Still, do an audit for all references to catalog functions.

Reply:
Will do